api/src/usersAvatar/services/usersAvatar.service.ts

```diff
@@ -39,23 +35,23 @@ export class UsersAvatarService {
         return this.usersAvatarRepository.save({ name, sourcePath, extension });
     }
 
-    async findOneByIdAndUserId(id: number, userId: number) {
-        const avatar = await this.usersAvatarRepository
+    async findOne(id: number) {
+        return this.usersAvatarRepository
             .findOneOrFail({ where: { id }, relations: { user: true } })
             .catch(error => {
                 if (error instanceof EntityNotFoundError) {
                     throw new NotFoundException();
                 }
                 throw error;
             });
+    }
 
-        if (!this.isDevelopment && avatar.user.id !== userId)
-            throw new ForbiddenException("You are not author of the avatar");
+    async delete(id: number, userId: number) {
+        const avatar = await this.findOne(id);
 
-        return avatar;
-    }
+        if (avatar.user.id !== userId)
+            throw new ForbiddenException("You aren't author of the avatar");
```

Comment:
It sounds weird, `author of the avatar`?

Reply:
ye, have you other idea?

Reply:
I know, what's wrong, i propose "... owner of the avatar", good?

Reply:
Better